app/views/search/_module_heading.html.erb

```diff
@@ -1,15 +1,9 @@
-<div class="row">
-  <div class="col-md-5">
-    <h2 class='result-set-heading'>
-      <%= I18n.t("#{service_name}_search.display_name") %>
+<div class="d-flex justify-content-between align-items-center">
+    <h2 class='result-set-heading h4'>
```

Comment:
The weight of the heading for the "SearchWorks" part of "SearchWorks Articles+" and "SearchWorks Catalog" appears to be about 300 in the Figma, whereas here it ends up being 500.

Reply:
Nice catch.  The font-size is h4, so I also had the weight for h4. I've added fw-light now.